machine/v1beta1/types_awsprovider.go

```diff
@@ -167,6 +167,13 @@ type EBSBlockDeviceSpec struct {
 	// it is not used in requests to create gp2, st1, sc1, or standard volumes.
 	// +optional
 	Iops *int64 `json:"iops,omitempty"`
+	// throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
```

Comment:
```suggestion
	// Throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
```
Not sure if lint happy with this :thinking:?

Reply:
Ahh thanks, that's a bit odd tho :D

Reply:
Fixed.